doc/conf.py

```diff
@@ -103,17 +103,18 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "python_docs_theme"
+html_theme = "furo"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {
-    "collapsiblesidebar": True,
-    "issues_url": "https://github.com/pycqa/pylint/issues/new",
-    "root_name": "PyCQA",
-    "root_url": "https://meta.pycqa.org/en/latest/",
-}
+# TO-DO: Disable thme options too see how Furo themes look in default config
```

Comment:
@gunungpw Would you like to do a follow-up to remove these options? 😄

Reply:
Okay 👍